.changeset/shy-rockets-melt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-graph': patch
+---
+
+Update the dependencyManager to include databases.
```

Comment:
@akphi  I think a change this small probably doesn't need a changeset log. Thoughts?